css/style.css

```
@@ -0,0 +1,215 @@
+/* 
+Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+Font sizes (px)
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+*/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.main-wrapper {
+  width: 160rem;
+  margin: 0 auto;
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.promo-header {
+  position: absolute;
+  display: flex;  
+  height: 12rem;  
+  align-items: center;
+  padding-left: 18rem;    
+  gap: 0.8rem;
+}
+
+.header-shadow {
+  width: 160rem;
+  position: absolute;  
+  height: 12rem;    
+  background: linear-gradient(to bottom, black -100%, transparent 90%);  
+}
```

Let's rename `promo-header` to `main-header` because it's doesn't relate to section "promo" but to whole page. Therefore:
```suggestion
.main-header:before {
  content: '',
  width: 100%;
  height: 100%;
  top: 0,
  left: 0,
  position: absolute;  
  background: linear-gradient(to bottom, black -100%, transparent 90%);  
}
```

---

I understood the principle of the pseudo-element. applied